server/fishtest/views.py

```diff
@@ -1082,7 +1103,12 @@ def tests_stop(request):
 def tests_approve(request):
     if not request.authenticated_userid:
         request.session.flash("Please login")
-        return HTTPFound(location=request.route_url("login"))
+        next_page = request.path
```

Comment:
I wonder if this code is reachable after exposing only the options to which the user is entitled.

Reply:
Well, in case an approver logged out from one tab and clicked approve on the other tab I guess that's the case.

Reply:
Already tried but I got "check_csrf_token(): Invalid token"

Reply:
Hmmm right, we require the token here which is removed if a user logged out I missed that.

Reply:
> Already tried but I got "check_csrf_token(): Invalid token", same error using the back button. I'm not able to trigger that condition.

I'm keeping the return though for security reasons if the csrf was hacked/broken somehow.